### PR TITLE
Added portfolio about and contact section with css style

### PR DESCRIPTION
**Pull request by Nasirkhan294**
Added portfolio section and stylized it.
Added about section and stylized it.
Added contact section and stylized it.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -42,6 +42,21 @@ header {
   padding: 15px 0;
 }
```
Comment (GedeonTS) [inline css/styles.css:43]: - Kindly make sure you have the border-bottom left radius like the one on the Figma design as we have to stick to it :recycle: 

Your design:
Screenshot from 2023-03-24 18-48-09

Figma:
Screenshot from 2023-03-24 18-53-58

File: index.html [hunk 2]
```diff
@@ -100,6 +106,221 @@ <h1>I'm Nasir Glad to see you!</h1>
...
+      <section id="contact">
+        <div class="container">
+          <h2>Contact me</h2>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it!
+          </p>
+          <form action="#">
+            <input type="text" placeholder="full name" class="contact-input" />
```
Comment (GedeonTS) [inline index.html:305]: - [OPTIONAL] Good job doing more than required for this milestone :clap:, Kindly reduce the border top left radius to the value provided on the Figma template to make it even better :sunglasses:

File: index.html [hunk 3]
```diff
@@ -100,6 +106,221 @@ <h1>I'm Nasir Glad to see you!</h1>
...
+              <h2>About Myself</h2>
+              <p>
+                I'm a software developer! I can help you build a project,
+                feature or website look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don't hesitate to contact me.
+              </p>
+            </div>
+            <!-- End hero main content section -->
+            <!-- Start hero social section -->
```
Comment (GedeonTS) [inline index.html:221]: - [ ] I suggest adding the border top right radius right as per the Figma design as we have to stick to it as much as possible :recycle: 

Your design:
Screenshot from 2023-03-24 19-00-43

Figma:

Screenshot from 2023-03-24 18-58-37

File: css/styles.css [hunk 4]
```diff
@@ -42,6 +42,21 @@ header {
...
+  border-radius: 8px;
+  cursor: pointer;
+  font-size: 1em;
+}
+
+.btn:hover {
+  background-color: #6070FF;
+  color: #fff;
+}
+
```
Comment (GedeonTS) [inline css/styles.css:59]: - Good job adding the hover effect, I suggest adding the active and enabled too as shown on the Figma design:
Screenshot from 2023-03-24 19-06-37

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @Nasirkhan294 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
 - No linter errors :+1: 
 - Good use of GitHub flow :+1: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lookingfor00 — APPROVED** [review]
# Approved 🟢
Hi @Nasirkhan294, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
allfather-approves-approved

## Highlights
- Clean code 🧼 👍 
- Nice design 🥇 
- Implemented all changes requested ♻️ 
- Professional documented project 📖 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@Trast00) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**